src/main/resources/liquibase/crop_changelog/19_5_1.xml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+	
+	<changeSet author="rowena" id="v19.5.1-1">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) > 0
+				FROM germplsm g
+						 INNER JOIN location l1
+						 INNER JOIN location l2
+				WHERE g.glocn = l1.locid AND l1.locid > l2.locid AND l1.lname = l2.lname AND l1.ltype = l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid = l1.snl1id
+              </sqlCheck>
+        </preConditions>
+        <comment> check if there are germplasm using duplicate location_id. Replace the duplicates with the locid of the minimum location ID of the duplicate</comment>
+		<sql dbms="mysql" splitStatements="true">
+	      	UPDATE germplsm g INNER JOIN location l1 INNER JOIN location l2
+			SET glocn = l2.locid
+			WHERE g.glocn = l1.locid AND l1.locid &lt; l2.locid AND l1.lname = l2.lname AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id;
```

Comment:
I notice the update statement use `&lt;` instead of ">" - which is used in the precondition